scripts/migrations.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+for MIGRATION in $(ls metadata_store_scripts/V*.sql)
+do
+  git checkout origin/main -- metadata_store_scripts/bootstrap.sql
+  script_version_and_comment=( $(basename -- "$MIGRATION" .sql ) )
+  echo "$script_version_and_comment"
+  lines_in_bootstrap=( $(grep "$script_version_and_comment" metadata_store_scripts/bootstrap.sql | wc -l) )
+  if [[ $lines_in_bootstrap -eq 2 ]]; then
+    # Nothing to do, assume this is correct
```

Comment:
[nit] Perhaps we should make the comment clearer as to why this is okay. If a `bootstrap.sql` file is properly configured, then the file should have exactly two lines containing the migration script's filename.